docs/examples/Sensor_Platform_Simulation.py

```diff
@@ -0,0 +1,380 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+Sensor Platform Simulation Example
+==================================
+This example looks at how platforms and sensors can be used within the Stone Soup simulation capability.
+"""
+
+# %%
+# Building a Simulated Sensor Platform
+# ------------------------------------
+# This example is going to look at how *Platforms* and *Sensors* can be combined with the simulation capabilites within
+# Stone Soup.
+#
+# The focus of this example is to show how to setup and configure simulations, as such the application of a tracker
+# will not be covered in detail. For more information about trackers and how to configure them review of the
+# tutorials and demonstrations is recommended.
+#
+# This example makes use of Stone Soup :class:`~.FixedPlatform` and :class:`~.Sensor` objects.
+#
+# In order to configure platforms, sensors and the simulation we will need to import some specific Stone Soup objects.
+# As these have been introduced in previous tutorials they are imported upfront. New functionality within this example
+# will be imported at the relevant point in order to draw attention to the new features.
+
+# Some general imports and set up
+from datetime import datetime
+from datetime import timedelta
+
+import numpy as np
+
+# Stone Soup imports:
+from stonesoup.types.state import State, GaussianState
+from stonesoup.types.array import StateVector, CovarianceMatrix
+from stonesoup.models.transition.linear import (
+    CombinedLinearGaussianTransitionModel, ConstantVelocity)
+from stonesoup.models.measurement.nonlinear import CartesianToElevationBearingRange
+from stonesoup.updater.kalman import UnscentedKalmanUpdater
+from stonesoup.predictor.kalman import UnscentedKalmanPredictor
+from stonesoup.deleter.time import UpdateTimeStepsDeleter
+from stonesoup.tracker.simple import MultiTargetTracker
+from matplotlib import pyplot as plt
+
+# Define the simulation start time
+start_time = datetime.now()
+
+# %%
+# Create a Platform
+# -----------------
+# The first element we need to create is a platform. For this first example we will build a static (or *fixed*) platform
+# which is located at the origin. For this example we are going to work in a 6-dimensional state space our platform will
+# have the following :math:`\vec{x}`.
```

Comment:
For consistency with tutorials and to give the impression we talk with one voice we should use :math:`\mathbf{x}` to describe the state.

Reply:
done